torch/testing/_internal/common_nn.py

```diff
@@ -2046,9 +2046,9 @@ def fractional_max_pool3d_test(test_case):
     ),
     dict(
         fullname='Conv3d_groups',
-        constructor=lambda: nn.Conv3d(4, 6, kernel_size=3, groups=2),
+        constructor=lambda: nn.Conv3d(2, 4, kernel_size=3, groups=2),
         cpp_constructor_args='torch::nn::Conv3dOptions(4, 6, 3).groups(2)',
-        input_size=(2, 4, 4, 5, 4),
+        input_size=(1, 2, 3, 3, 3),
```

Comment:
@zasdfgbnm , looks like this change causes CuDNN v8 to fail with:
```
RuntimeError: cuDNN error: CUDNN_STATUS_NOT_SUPPORTED. This error may appear if you passed in a non-contiguous input.
Exception raised from operator() at /var/lib/jenkins/workspace/aten/src/ATen/native/cudnn/Conv.cpp:845 
```
see https://app.circleci.com/pipelines/github/pytorch/pytorch/188319/workflows/cf1b0d7c-66d5-4ba1-8d70-7d069999676f/jobs/6137541/tests